src/client.js

```diff
@@ -60,3 +61,24 @@ function refresh_token (_refresh_token) {
             return result;
         });
 }
+
+function revoke_token (token, client_id, type = '') {
+    const scopes_string = cache.find_key('client', {token});
+    const payload = {
+        client_id: client_id,
+        type: type,
+    };
+
+    return cudl.post
+        .to(config.base_url + config.path + '/oauth/revoke')
+        .send(payload)
```

Comment:
We're missing the header? Access-token that will be revoked needs to be in the header

Reply:
Thank you for noticing this requirement. I have already updated this on [commit fec473](https://github.com/anyTV/freedom-accounts-util/pull/10/commits/fec47360ba87c8875a76960c29a536e6ed8c239d). Kindly review it. :smile: